editions/test/tiddlers/tests/data/transclude/Parameterised-Conditional-Parameters-Refreshed.tid

```diff
@@ -0,0 +1,39 @@
+title: Transclude/Parameterised/ConditionalParameters/Refreshed
+description: Parameterised transclusion when conditional parameters are changed by a refresh
+type: text/vnd.tiddlywiki-multiple
+tags: [[$:/tags/wiki-test-spec]]
+
+title: Output
+
+\procedure elephant(filler)
+\whitespace trim
+<$let name={{Canary}}>
+<% if [<name>match[yes]] %>
+<$parameters A=defaultA>
+A/<$text text=<<filler>>/>/<$text text=<<A>>/>/<$text text=<<B>>/>
+</$parameters>
+<% else %>
+<$parameters B=defaultB>
+B/<$text text=<<filler>>/>/<$text text=<<A>>/>/<$text text=<<B>>/>
+</$parameters>
+<% endif %>
+\end elephant
+\whitespace trim
```

Comment:
May be this one needs to be at the top of the tiddler

Reply:
Yes, there has to be a whitespace trim before the procedure definitions in order for it to be inherited by the procedures.